### PR TITLE
Add basic proposal details page

### PR DESCRIPTION
**Pull request by csillag**
Design is [here](https://www.figma.com/file/8dwSyYOOm3vlgbvxPGT29p/Block-Explorer-(Post-Launch)?type=design&node-id=8960-67644&mode=design&t=ryRSEAVnWV2RwqbZ-0).

This PR only handles the routing, the various error corner cases, and the main card.

Also, there is some data that is part of the design, but not yet available for Nexus. We will add that later.

Also, link to the proposal detail pages from the proposal list.

### CODE REVIEW

File: src/app/pages/ProposalDetailsPage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+export const ProposalDetailView: FC<{
+  proposal: Proposal
+  isLoading?: boolean
+  showLayer?: boolean
+  standalone?: boolean
+}> = ({ proposal, isLoading, showLayer = false, standalone = false }) => {
+  const { t } = useTranslation()
+  const { isMobile } = useScreenSize()
+  if (isLoading) return <TextSkeleton numberOfRows={7} />
+  console.log('loaded proposal', proposal)
```
Comment (lukaw3d) [inline src/app/pages/ProposalDetailsPage/index.tsx:48]: TODO?

File: src/oasis-nexus/api.ts [hunk 2]
```diff
@@ -822,6 +827,31 @@ export const useGetConsensusProposals: typeof generated.useGetConsensusProposals
...
+    ...options,
+    request: {
+      ...options?.request,
+      transformResponse: [
+        ...arrayify(axios.defaults.transformResponse),
+        (data: generated.Proposal, headers, status) => {
+          if (status !== 200) return data
+          return {
+            ...data,
+            network,
```
Comment (lukaw3d) [inline src/oasis-nexus/api.ts:845]: https://github.com/oasisprotocol/explorer/pull/1192#discussion_r1470516266

**Review by lukaw3d — APPROVED** [review]

File: src/app/pages/ProposalDetailsPage/index.tsx (hunk 1 above)
Comment (csillag) [reply src/app/pages/ProposalDetailsPage/index.tsx:48]: Removed

File: src/oasis-nexus/api.ts (hunk 2 above)
Comment (csillag) [reply src/oasis-nexus/api.ts:846]: Yes, adding the consensus layer here now.

**Pull request merged**